src/models/servercountrymodel.h

```diff
@@ -74,10 +82,14 @@ class ServerCountryModel final : public QAbstractListModel {
 
   QVariant data(const QModelIndex& index, int role) const override;
 
+ signals:
+  void changed();
+
  private:
   [[nodiscard]] bool fromJsonInternal(const QByteArray& data);
 
   void sortCountries();
+  int cityConnectionScore(const ServerCity& city) const;
```

Comment:
ServerConnectionScore?

Reply:
In this case we are actually scoring the city and not any one particular server at that location, so I feel that calling this a `cityConnectionScore` is slightly more accurate.